tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -5007,7 +5043,19 @@ <h2>Show similarity to selected datapoint</h2>
         chart.seriesWithoutTooltips = thresholdSeries;
         chart.colorScale.domain(allSeries);
         chart.setVisibleSeries(allSeries);
-        return chart;
+        holder.appendChild(chart);
+
+        const yLabel = document.createElement('div');
+        yLabel.classList.add('pd-y-label');
+        yLabel.innerText = this.isRegression_(this.modelType) ?
+          'Inference value' : 'Inference score';
```

Comment:
+2 whitespace if you are following Google JS Style guide.

Reply:
done in both places